api/reference/users.adoc

```diff
@@ -148,7 +148,7 @@ POST /api/users/:id
 {
   "action" : "set_current_group",
   "resource" : {
```

Comment:
This bit is apparently wrong.

This should be:

```
{
  "action" : "set_current_group",
  "current_group": { "id": "12" },
}
```

(gitter :point_up: [July 26, 2019 8:19 AM](https://gitter.im/ManageIQ/api?at=5d3ab7a2f4c906673aef1b4e))

Reply:
@himdel Thank you. Good catch. I incorrectly tested with "action" : "edit"

Fixing now.